SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialLoadBalancingHandler.cpp

```diff
@@ -0,0 +1,125 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "Utils/SpatialLoadBalancingHandler.h"
+
+#include "EngineClasses/SpatialActorChannel.h"
+#include "EngineClasses/SpatialPackageMapClient.h"
+#include "LoadBalancing/AbstractLBStrategy.h"
+#include "LoadBalancing/OwnershipLockingPolicy.h"
+#include "Interop/SpatialSender.h"
+#include "Schema/SpatialDebugging.h"
+
+FSpatialLoadBalancingHandler::FSpatialLoadBalancingHandler(USpatialNetDriver* InNetDriver)
+	: NetDriver(InNetDriver)
+{
+
+}
+
+FSpatialLoadBalancingHandler::ProcessActorResult FSpatialLoadBalancingHandler::ProcessSingleActor(AActor* Actor, AActor*& OutNetOwner, VirtualWorkerId& OutWorkerId)
```

Comment:
Can this be renamed? We have `ProcessSingleActor` and `ProcessMigrations` which relate to two different parts of the flow? Maybe `EvaluateActorForMigration`?